zeppelin-web/src/app/jobmanager/jobmanager.filter.js

```diff
@@ -38,6 +39,17 @@
         });
       }
 
+      if (isSortByAsc === true) {
+        filterItems = _.sortBy(filterItems, function(sortItem) {
+          return sortItem.noteName;
+        });
+      } else {
+        filterItems = _.sortBy(filterItems, function(sortItem) {
```

Comment:
I think the function '_.orderBy()' is better. It allows specifying the sort orders.
Please refer [https://lodash.com/docs/4.17.2#orderBy](url)